packages/valory/skills/decision_maker_abci/handlers.py

```diff
@@ -48,4 +50,34 @@
 LedgerApiHandler = BaseLedgerApiHandler
 ContractApiHandler = BaseContractApiHandler
 TendermintHandler = BaseTendermintHandler
-IpfsHandler = BaseIpfsHandler
+
+
+class IpfsHandler(AbstractResponseHandler):
```

Comment:
you could just use the default IPFS handler, no?

Reply:
Correct. Will be addressed in #170.